markdown/org/docs/patterns/lucy/en.md

```diff
@@ -2,5 +2,7 @@
 title: "Lucy tie-on pocket"
 ---
 
+This pattern comes with three design options: 
```

Comment:
I don't believe this is necessary. 
The `<PatternDocs pattern='lucy' />` already displays the options and titles them as well.

Reply:
Ive removed the extra description